doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
+aliases: "/telemetry/cli"
+---
+
+{{% tts %}}\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used. Telemetry collection is enabled by default and does not collect personally identifiable information. The following section section explains what information is collected and how it is anonymised.
+
+## Identification
+
+When the CLI starts for the first time on a new machine or upon clearing the system cache, it creates a file on the system's cache folder with the following data:
+
+```yaml
+uid: 00000000-0000-0000-0000-000000000000
+last_sent: 2023-08-02T16:45:53.742417-03:00
+```
+
+The `uid` value is an UUID generated according to the specifications made on [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122) and [DCE 1.1](https://pubs.opengroup.org/onlinepubs/9696989899/toc.htm).
+
+## Types of telemetry data
+
+Below is an JSON object with an example of a full telemetry data message that is sent from the CLI:
+
+```json
+{
+  "uid": "00000000-0000-0000-0000-000000000000",
+  "os": {
+    "operating_system": "linux",
+    "arch": "amd64",
+    "binary_version": "3.27.0-dev",
+    "golang_version": " go1.20.7"
+  },
+  "cli": {}, // An empty object for this field indicates that this data was sent by a CLI instance
+  "entities_count": null
+}
+```
+
+{{< note >}}
+All new data collected will be added below with the respective version in which the field was introduced. 
```

Comment:
I don't understand this; below what? respective version of what? which field? Try to be more specific

Reply:
Removed the idea of listing what field was added in which version.